jekyll/_cci2/monitoring.adoc

```diff
@@ -205,7 +269,114 @@ image::custom_metrics.png[Custom Metrics]
+  urls = ["https://us-central1-1.gcp.cloud2.influxdata.com"]
+  token = "YOUR_TOKEN_HERE"
+  organization = "circle@example.com"
+  bucket = "circleci"
+```
+
+===== Scenario 2: Record all metrics to Datadog
+{:.no_toc}
+
+The standard configuration handles only selected metrics, and there are many metrics discarded by Telegraf. If you want to receive this discarded, sophisticated data, such as JVM stats and per-container CPU usage, you can keep all received metrics by removing namepass filter. This example also illustrates how to configure metrics emission to Datadog. As discussed above, you need manual configuration for outputs to Datadog regardless of configurations on Replicated.
+
+CAUTION: This scenario leads to very large amounts of data.
+
+```
+[[inputs.statsd]]
+  service_address = ":8125"
+  parse_data_dog_tags = true
+  metric_separator = "."
+
+[[outputs.datadog]]
+  apikey = 'YOUR_API_KEY_HERE'
+```
+
+===== Scenario 3: Send limited metrics to CloudWatch
+{:.no_toc}
+
+AWS charges fees for CloudWatch per series of scalar (i.e. at the level of "mean" or "sum"). Since multiple fields (e.g. mean, max, min and sum) are calculated for each metrics key (e.g. `circle.run-queue.latency`) and some fields can be redundant, you might want to select which fields are send to CloudWatch. This can be achieved by configuring `\[[outputs.cloudwatch]]` with `fieldpass`. You also may declare `\[[outputs.cloudwatch]]` multiple times to pick up multiple metrics, as illustrated below.
+
+```
+[[inputs.statsd]]
+  # Accept all metrics at input level to 1) enable output configurations without thinking of inputs, and to 2) dump discarded metrics to stdout just in case.
+  service_address = ":8125"
+  parse_data_dog_tags = true
+  metric_separator = "."
+
+[[outputs.cloudwatch]]
+    # Fill in these two variables if you need to access CloudWatch with an IAM User, not an IAM Role attached to your Services box
+    # access_key = 'ACCESS'
+    # secret_key = 'SECRET'
+
+    # Specify region for CloudWatch
+    region = 'ap-northeast-1'
+    # Specify namespace for easier monitoring
+    namespace = 'my-circleci-server'
+
+    # Name of metrics key to record
+    namepass = ['circle.run-queue.latency']
+    # Name of metrics field to record; key and field are delimited by an underscore (_)
+    fieldpass = ['mean']
+
+[[outputs.cloudwatch]]
+    # Outputs can be specified multiple times.
+
+    # Fill in these two variables if you need to access CloudWatch with an IAM User, not an IAM Role attached to your Services box
+    # access_key = 'ACCESS'
+    # secret_key = 'SECRET'
+
+    # Specify region for CloudWatch
+    region = 'ap-northeast-1'
+    # Specify namespace for easier monitoring
+    namespace = 'my-circleci-server'
+
+    # Name of metrics key to record
+    namepass = ['mem']
+    # Name of metrics field to record; key and field are delimited by an underscore (_)
+    fieldpass = ['available_percent']
+```
 
 === Configuring Custom Metrics
```

Comment:
@makotom with all this new really great info you've given us, I'm not sure if we still need this section? Or maybe it just needs a better heading?